### DISCUSSION TITLE
Systray icon adjustment

### DISCUSSION THREAD

**Discussion by 4164696f73** [fcitx/fcitx5#406] [General]
This might be a new feature, because I just noticed it today, but can you make the systray icon not change to language text when selecting a text field? For example, when I'm not in any text editor, chat or any kind of text field, the systray icon is a keyboard, however when I select any kind of text field, the keyboard changes to my language, for example "en", meaning I type in english. I'd like to keep it as a keyboard at all times, but I can not find anything that would do what I want.

Sent in discussions because it's more of a "wanna have" than an issue. Thank you in advance.

**Comment by wengxt** [comment]
it will display the layout name when your config has more than one layout configured, is that the case?

**Reply by 4164696f73** [reply]
Yeah, exactly. I'd like it to not ever do that if I choose so. No particular reason for that other than that I don't like it to change to text.
